core/src/main/java/bisq/core/api/CorePaymentAccountsService.java

```diff
@@ -135,6 +134,7 @@ synchronized PaymentAccount createCryptoCurrencyPaymentAccount(String accountNam
                                                       String address,
                                                       boolean tradeInstant) {
         accountService.checkAccountOpen();
+        verifyAccountNameUniqueness(accountName);
```

Comment:
Please rename to `verifyAccountNameUnique` for brevity.

Reply:
done